docs/en/guides/triton-inference-server.md

```diff
@@ -153,14 +153,18 @@ for _ in range(10):
 
 Then run inference using the Triton Server model:
 
+!!! warning "Model metadata"
+
+    If you're using `Segment`, `Pose`, or `OBB` model, it is necessary to pass the `data.yaml` file associated with the model during prediction to prevent errors. It is also advisable to do the same for `Detect` and `Classify` models so that the inference can use correct class names from the metadata.
+
 ```python
 from ultralytics import YOLO
 
 # Load the Triton Server model
 model = YOLO("http://localhost:8000/yolo", task="detect")
 
-# Run inference on the server
-results = model("path/to/image.jpg")
+# Provide the data.yaml file associated with model during prediction.
+results = model("path/to/image.jpg", data="coco8.yaml")
```

Comment:
Is this related to onnxslim, I just added metadata.

Reply:
@inisis was this a recent bug fix? Can you implement a minimum onnxslim version PR here to make sure users use at least this version?

Reply:
@glenn-jocher No, metadata was added to onnxslim three weekes ago in version 0.1.39 and now it's 0.1.42

Reply:
@inisis It's not ONNX related. It's due to how Triton Inference Server works. We don't have access to metadata of the model since the model is loaded in the server and not in Ultralytics.